pkg/roachpb/data.go

```diff
@@ -2783,3 +2783,10 @@ func (tid *TenantID) UnmarshalYAML(unmarshal func(interface{}) error) error {
 		return unmarshal(tid)
 	}
 }
+
+var _ redact.SafeFormatter = LeaseAcquisitionType(0)
+
+// Safeformat implements the redact.SafeFormatter interface.
```

Comment:
Instead of implementing the `redact.SafeFormat` interface and providing an implementation for this method, could you have implemented the `redact.SafeValue` interace?

```go
func (LeaseAcquisitionType) SafeValue() {}
```

You'd then also need to update `pkg/testutils/lint/passes/redactcheck/redactcheck.go`.

Reply:
TIL. Made this change. Thanks!